### PR TITLE
translation for issue #1

### PR DESCRIPTION
**Pull request by rkno82**
My first pull request. :)

### CODE REVIEW

**Comment by rkno82** [comment]
#1

File: README.md [hunk 1]
```diff
@@ -12,40 +12,58 @@
...
+- The templates, ssh_config.erb and sshd_config.erb, can be used without puppet. There is no deep ruby or puppet knowhow needed.
+- The disclaimer, that those files are managed by puppet, should be removed.
+
+### Verification
+
+- The tests (spec/acceptance) can be used to verify the hardening, even without using the module on your servers.
+
+## Reference
+
+- The hardening requirements are from Telekom Security, Privacy and Security Assessment - 3_04_Secure_Shell_(SSH). Last verified on 2018-07-31.
```
Comment (rndmh3ro) [inline README.md]: Can you add a link to the psa here? (https://www.telekom.com/psa).

Comment (rkno82) [reply README.md]: That cannot be resolved externally?

**Comment by rkno82** [comment]
@schurzi Please have a look into that.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/T-Systems-MMS/puppet-secc_sshd.svg?branch=master)](https://travis-ci.org/T-Systems-MMS/puppet-secc_sshd)
 
-####Table of Contents
+## Table of Contents
 
 1. [Overview](#overview)
```
Comment (schurzi) [inline README.md]: TOC needs update

File: README.md [hunk 3]
```diff
@@ -12,40 +12,58 @@
...
+
+- SoC Requirements 19, 22 are fulfilled without restrictions.
+- SoC Requirement 20 (security of the private key) cannot be verified by ssh client or daemon.
+- SoC Requirement 21 (usage of the ssh auth-agent) cannot be verified by ssh client or daemon.
+
+## Usage
+
+- Either include the module via git or puppetforge. (T-Systems is using r10k)
+
+### Usage ohne Puppet
```
Comment (schurzi) [inline README.md]: `ohne`?

Comment (rkno82) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (rkno82) [reply README.md]: missing content removed, please check again
TOC is working!

**Comment by schurzi** [comment]
Build is currently broken due to a dependency update to beaker 4 (they removed all hypervisors from the implicit dependency list)

**Pull request merged**